src/multiselectMixin.js

```diff
@@ -688,7 +688,7 @@ export default {
       this.isOpen = false
       /* istanbul ignore else  */
       if (this.searchable) {
-        if (typeof this.$refs.search !== 'undefined') this.$refs.search.blur()
+        if (this.$refs.search !== null && typeof this.$refs.search !== 'undefined') this.$refs.search.blur()
```

Comment:
Why not just simplify to
`this.$refs.search && this.$refs.search.blur()`
?
I mean make the same as [here](https://github.com/shentao/vue-multiselect/blob/e35dd377a4cf41732a10c27ee0adf969af12d68e/src/multiselectMixin.js#L674)

Reply:
@shentao can we please have some fix approved?